.circleci/verbatim-sources/job-specs/job-specs-custom.yml

```diff
@@ -89,9 +89,8 @@
           set -ex
           export COMMIT_DOCKER_IMAGE=${DOCKER_IMAGE}:build-${DOCKER_TAG}-${CIRCLE_SHA1}
           echo "DOCKER_IMAGE: "${COMMIT_DOCKER_IMAGE}
-          # turn v1.12.0rc3 into 1.12.0
-          tag=$(echo $CIRCLE_TAG | sed -e 's/v*\([0-9.]*\).*/\1/')
-          tag=${CIRCLE_TAG:1:5}
+          # turn v1.12.0rc3 into 1.12
+          tag=$(echo $CIRCLE_TAG | sed -e 's/v*\([0-9]*\.[0-9]*\).*/\1/')
```

Comment:
I'm not sure why the `tag=${CIRCLE_TAG:1:5}` is still here, but I think the two appearances should be synced.